docs/README.template.md

```diff
@@ -23,30 +23,52 @@ A low-level JS <a href="https://interledger.org">Interledger</a> sender/receiver
 [snyk-image]: https://snyk.io/test/npm/ilp/badge.svg
 [snyk-url]: https://snyk.io/test/npm/ilp
 
-This is a low-level interface to ILP, largely intended for building ILP into other [Application layer](https://github.com/interledger/rfcs/tree/master/0001-interledger-architecture) protocols.
-
-#### The ILP Client includes:
+#### The ILP module includes:
 
 * [Interledger Payment Request (IPR)](#interledger-payment-request-ipr-transport-protocol) Transport Protocol, an interactive protocol in which the receiver specifies the payment details, including the condition
 * [Pre-Shared Key (PSK)](#pre-shared-key-psk-transport-protocol) Transport Protocol, a non-interactive protocol in which the sender creates the payment details and uses a shared secret to generate the conditions
+* [Simple Payment Setup Protocol (SPSP)](#simple-payment-setup-protocol-spsp), a higher level interface for sending ILP payments, which requires the receiver to have an SPSP server.
 * Interledger Quoting and the ability to send through multiple ledger types using [Ledger Plugins](https://github.com/interledgerjs?utf8=✓&q=ilp-plugin)
 
-#### The ILP Client does **not** handle:
-
-* Account discovery
-* Amount negotiation
-* Communication of requests from recipient to sender
-
 ## Installation
 
 `npm install --save ilp ilp-plugin-bells`
 
 *Note that [ledger plugins](https://www.npmjs.com/search?q=ilp-plugin) must be installed alongside this module*
 
+## [Simple Payment Setup Protocol (SPSP)](https://github.com/interledger/rfcs/blob/master/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md)
+
+If you are sending to an SPSP receiver with a `user@example.com` identifier, the SPSP module
+provides a high-level interface:
+
+```js
+'use strict'
+
+const co = require('co')
+const SPSP = require('ilp').SPSP
+const FiveBellsLedgerPlugin = require('ilp-plugin-bells')
+
+const plugin = new FiveBellsLedgerPlugin({
+  account: 'https://red.ilpdemo.org/ledger/accounts/alice',
+  password: 'alice'
+})
+
+co(function * () {
```

Comment:
Since async/await has landed in Node.js I think it would be fine to use it in the example.

I do realize that not all JavaScript developers are familiar with it yet, but it's going to spread quickly and we want this to look modern. Maybe include a link to a page that explains async/await somewhere.